jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -797,6 +803,34 @@ public StringBuffer getRequestURL()
                 .port(getServerPort())
                 .asString());
         }
+
+        @Override
+        public Object getAttribute(String name)
+        {
+            return switch (name)
+            {
+                case ERROR_REQUEST_URI -> _httpServletRequest.getRequestURI();
```

Comment:
You could call `getRequest` and cast it to `HttpServletRequest` instead of introducing the `_httpServletRequest` field.